cmd/kube-scheduler/app/server.go

```diff
@@ -233,7 +232,6 @@ func installMetricHandler(pathRecorderMux *mux.PathRecorderMux) {
 	defaultMetricsHandler := legacyregistry.Handler().ServeHTTP
 	pathRecorderMux.HandleFunc("/metrics", func(w http.ResponseWriter, req *http.Request) {
 		if req.Method == "DELETE" {
-			metrics.Reset()
```

Comment:
why is this removed?

Reply:
I removed DeprecatedSchedulingDuration from metrics.go and because of that removed Reset() as well because it was calling DeprecatedSchedulingDuration.Reset()

Reply:
ah, sounds good then